src/app/learn/reference/cli-commands/install/index.html

```diff
@@ -0,0 +1,8 @@
+<stache
+  pageTitle="install">
+
+  <p>
+    <stache-code>skyux install</stache-code> deletes the <stache-code>node_modules</stache-code> folder and runs <stache-code>npm install</stache-code>. This clears the existing dependencies in the <stache-code>node_modules</stache-code> folder and ensures that all SKY UX dependencies are consistent and up to date based on your <stache-code>package.json</stache-code> file.
```

Comment:
Maybe something like, “...SKY UX dependencies are consistent with your `package.json` file.“? The feature essentially deletes the node_modules folder and reinstalls all dependencies as they are defined in package.json.

Reply:
I'll make the change.